docs/swagger.yaml

```diff
@@ -911,5 +1020,9 @@ components:
         email:
           type: string
           description: 'Example: "user@example.com"'
+        userInfoSub:
```

Comment:
Should remove this for now - it is hidden for a while to assess how it works for KomplettNettbutikk.